DPGAnalysis/Skims/python/WElectronSkim_cff.py

```diff
@@ -104,10 +104,22 @@
     hltTags = cms.untracked.string( HLTPath ),
     triggerEventTag = cms.untracked.InputTag("hltTriggerSummaryAOD","",HLTProcessName),
     triggerResultsTag = cms.untracked.InputTag("TriggerResults","",HLTProcessName),
-    stageL1Trigger = cms.uint32(1)
+    stageL1Trigger = cms.uint32(1),
+    # Stage-1 L1T inputs
+    l1GtRecordInputTag = cms.InputTag('gtDigis'),
+    l1GtReadoutRecordInputTag = cms.InputTag('gtDigis'),
 )
 from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
-stage2L1Trigger.toModify(PassingHLT, stageL1Trigger = 2)
+#stage2L1Trigger.toModify(PassingHLT, stageL1Trigger = 2)
```

Comment:
You don't need to keep the old one. Git already keeps track of modification.

Reply:
Oh OK, I didn't notice that, thanks! Will remove.